pkg/controller/external_nofork.go

```diff
@@ -286,6 +335,29 @@ func (n *noForkExternal) getResourceDataDiff(ctx context.Context, s *tf.Instance
 			return nil, errors.Wrap(err, "failed to compute the customized terraform.InstanceDiff")
 		}
 	}
+	if !instanceDiff.Empty() && resourceExists {
+		for _, keyToIgnore := range n.initProviderExclusiveParamKeys {
+			for attributeKey := range instanceDiff.Attributes {
+				if keyToIgnore != attributeKey {
+					continue
+				}
+
+				if err := deleteInstanceDiffAttribute(instanceDiff, keyToIgnore); err != nil {
+					return nil, errors.Wrapf(err, "cannot delete key (%v) from instance diff.", keyToIgnore)
+				}
+
+				keyComponents := strings.Split(keyToIgnore, ".")
+				if keyComponents[0] != "tags" {
+					continue
+				}
+				keyComponents[0] = "tags_all"
```

Comment:
Should we also check if the resource Terraform schema contains `tags_all`? I believe, if it does not contain, then there will be no corresponding diff attributes and the `deleteInstanceDiffAttribute` should be a noop, so I believe we are fine. Is this correct, what do you think?